hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -218,9 +218,11 @@ public synchronized void close() {
       channel.shutdownNow();
       try {
         channel.awaitTermination(60, TimeUnit.MINUTES);
-      } catch (Exception e) {
+      } catch (InterruptedException e) {
         LOG.error("Unexpected exception while waiting for channel termination",
```

Comment:
can you update this log text as well?

Reply:
Of course,I will update it now.

Reply:
Thanks @jojochuang  for the review, will this look okay?

Reply:
ok this is fine.